test/ClangImporter/objc_parse.swift

```diff
@@ -310,7 +310,7 @@ class NSObjectable : NSObjectProtocol {
 // Properties with custom accessors
 func customAccessors(_ hive: Hive, bee: Bee) {
   markUsed(hive.isMakingHoney)
-  markUsed(hive.makingHoney()) // expected-error{{cannot call value of non-function type 'Bool'}}{{28-30=}}
+  markUsed(hive.makingHoney()) // expected-error{{cannot call value of non-function type '@lvalue Bool'}}{{28-30=}}
```

Comment:
Ah, sorry, it looks like it should be using `fnType` instead of `CS->getType(fnType)` in for that message, I'll change that.